### ISSUE TITLE
Can't add Disk Mounter applet to a panel

### ISSUE THREAD

**Issue by flexiondotorg** [mate-desktop/mate-applets#251]
#### Expected behaviour

Right click a panel, find Disk Mounter, click Add and the applet should be added to the panel. 

#### Actual behaviour

Right click a panel, find Disk Mounter, clicking Add appears to do nothing.

#### Steps to reproduce the behaviour

See *Actual behaviour* above.

#### MATE general version

  * MATE Desktop 1.18

#### Package version

  * mate-applets 1.18.1-0ubuntu1

#### Linux Distribution

  * Ubuntu MATE 17.10 daily.

**Comment by monsta** [comment]
I know this issue... it's actually added, but until you'll plug in some drive, it won't show anything.

**Comment by marosg42** [comment]
I decided to give it a try, to change the code. 
What I achieved is - if applet is added and there are no drives to show, it will create an empty unclickable button...
I have all the logic working (when media is inserted, rejected etc), but I would need a help with actual icon. As it is now, icon is empty, it means it is not visible unless you hover over it with a mouse.
This is the actual code from drive-button.c

```
    if (!self->volume && !self->mount) {
	    if (gtk_bin_get_child (GTK_BIN (self)) != NULL)
	      gtk_image_set_from_pixbuf (GTK_IMAGE (gtk_bin_get_child (GTK_BIN (self))), NULL);
	    return FALSE;
    }

```
Could somebody give me a working code which would put there some default icon or maybe DiskMount icon?
